### ISSUE TITLE
Escape button doesn't seem to work on mobile(android)

### ISSUE THREAD

**Issue by RedlineTriad** [unitystation/unitystation#1910]
# Description
Nothing happens when the escape button is pressed on the mobile version (android)

## Category
BUG

## Detailed Bug Report
[Video of it not happening](https://youtu.be/AF7F7mc7LwU)

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1. Use the mobile version (android)
2. Start game
3. Select any job
4. Press the escape button on the right
5. Notice the lack of happening

**Comment by fomalsd** [comment]
yeah, Z, X and Enter from virtual gamepad aren't working on Android for some reason, too (but F5 seems to work)
